src/index.scss

```diff
@@ -149,3 +149,61 @@ img {
 a {
   transition: all 0.3s;
 }
+
+.container {
+  width: 100%;
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+@media (min-width: 576px) {
+  .container {
+    max-width: 540px;
+    padding-right: 0;
+    padding-left: 0;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    max-width: 720px;
+  }
+}
+
+@media (min-width: 992px) {
+  .container {
+    max-width: 960px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .container {
+    max-width: 1170px;
+  }
+}
+
+@media (min-width: 1350px) {
+  .container {
+    max-width: 1320px;
+  }
+}
+
+@media (min-width: 1440px) {
+  .container {
+    max-width: 1400px;
+  }
+}
+
+@media (min-width: 1600px) {
+  .container {
+    max-width: 1570px;
+  }
+}
+
+@media (min-width: 1920px) {
+  .container {
+    max-width: 1890px;
+  }
+}
```

Comment:
I'd say that it's better to have less media query selectors and just let the layout be more fluid. Of course you need one for the mobile vs. desktop cutoff.